### DISCUSSION TITLE
Publish Button Greyed Out

### DISCUSSION THREAD

**Discussion by COMPUMEC** [code4recovery/12-step-meeting-list#863] [Q&A] [answered]
In trying to create a new meeting the publish button stays grayed out and the map window does not populate. Here is a video showing the problem.

https://www.dropbox.com/s/hng92ogdso5zgea/Add%20New%20Meeting%20%E2%80%B9%20Central%20Service%20Office%20%E2%80%94%20WordPress%20-%20Google%20Chrome%202022-07-12%2014-22-39.mp4?dl=0

**Comment by Grateful-Viv** [comment]
There's a space before the first character in Region. What happens if you take that out?

**Reply by COMPUMEC** [reply]
I double checked and there is no space. All regions appear that way.
We are not the only ones having this same problem.

theronb wrote:

Bob - I have the same problem, but please note that TSML support has moved to GitHub: https://github.com/code4recovery/12-step-meeting-list/discussions

Post Link: https://wordpress.org/support/topic/cant-create-new-meeting/#post-15819473
-----------
You are receiving this email because you subscribed to a forum topic.

Log in and visit the topic to reply to the topic or unsubscribe from these emails. Note that replying to this email has no effect.

**Reply by COMPUMEC** [reply]
There is another problem possibly connected. 
When sorting meetings by region in the control panel for "Portland" it displays only a total of 40 meetings when on the website there are 3 or 4 times that many.
Watch video
https://www.dropbox.com/s/mwggvov7mw5e6m1/Dashboard%20%E2%80%B9%20Central%20Service%20Office%20%E2%80%94%20WordPress%20-%20Google%20Chrome%202022-07-12%2018-45-03.mp4?dl=0

**Comment by digimech** [comment]
I am having this issue too - publish is either grayed out or if available won't actually publish the meeting post (stays as draft)

**Comment by brianw-area24** [comment]
When this happens, something you might try is to remove the ", USA" at the end off the address, then move to a different field on the form. This should engage cause the form to geocode the address again, and will likely add the ", USA" back in. It should also cause the publish button to be clickable again.

**Reply by BDS419** [reply]
I am unable to edit an existing meeting or add a new meeting. I believe it may be related to the mapping function. Any other suggestions I should try? I can't even change the notes on an existing meeting. It just won't update. Please help. Thank you.

**Reply by COMPUMEC** [reply]
I tried what you suggested and there is no change.

There is another problem possibly connected.
When sorting meetings by region in the control panel for "Portland" it displays only a total of 40 meetings when on the website there are 3 or 4 times that many.

Watch video
https://www.dropbox.com/s/mwggvov7mw5e6m1/Dashboard%20%E2%80%B9%20Central%20Service%20Office%20%E2%80%94%20WordPress%20-%20Google%20Chrome%202022-07-12%2018-45-03.mp4?dl=0

**Comment by lionssharedigital** [comment]
Nashville and Austin are having the same issue. 
Reported here: https://github.com/code4recovery/12-step-meeting-list/discussions/833#discussioncomment-3140313

**Comment by district6wc** [comment]
District 6 in North Carolina is having this same issue, we can't update changes to location or zoom links. Once we add content to those fields the publish/update button grays out.

**Comment by tim-rohrer** [comment, marked as answer]
It turns out, we (Code for Recovery) had a problem with our Google account, which has now been resolved. 

You should all be able to return to:

MapBox token for mapping (or you can continue to use your own Google mapping key if you prefer).
Legacy for geocoding (or you can use the Beta, but it will be replaced in the (not too distant, perhaps) future).

Apologies for the interruption of service. I know it is frustrating.

**Reply by tim-rohrer** [reply]
For the OP, if your issue is not resolved, please reply to this thread.

For others, if you are experiencing an issue, please open a new Discussion topic as I suspect something else is going one.

**Reply by COMPUMEC** [reply]
Thank you.
